src/actionCreators/index.js

```diff
@@ -37,18 +38,29 @@ const apiPollTimeout = 3000;
 const maxStatusCheckCount = (120 * 1000) / apiPollTimeout;
 let statusCheckCount = 0;
 
-export const createSigningJob = files => dispatch => {
+const authenticationHeaderVaule = () => {
+  try {
+    const token = JSON.parse(localStorage.getItem('auth')).id_token.bearer;
```

Comment:
just as a warning,  in version 0.0.7 of the @bcgov/common-web-utils. Any tokens that are saved in local storage have their names camelized (id_token -> idToken). That would cause this code to break. A few work arounds **if you update to 0.0.7 or greater**: 

```javascript
  // work around 1
 // import your implicit auth instance into action creator
 //  replace code with 
const token = implicitAuthInstance.idToken.bearer

// work around 2
const token = JSON.parse(localStorage.getItem('auth').idToken.bearer;
```

Reply:
Updated. Changed to work around 1.